src/main/kotlin/calculator/view/ConsoleInput.kt

```diff
@@ -0,0 +1,14 @@
+package calculator.view
+
+class ConsoleInput {
+
+    companion object {
+        private const val DELIMITER = " "
+    }
+
+    fun readExpression(): List<String> {
+        return readLine()?.also { if (it.isBlank()) throw IllegalArgumentException("empty string is not allowed") }
+            ?.run { split(DELIMITER) }
+            ?: throw IllegalArgumentException("null is not allowed")
+    }
```

Comment:
`readln`을 사용하면 `nullable`을 제거할 수 있고, `ifBlank`를 사용하면 `also`도 제거할 수 있어요!
```suggestion
        return readln().ifBlank {
            throw IllegalArgumentException("empty string is not allowed")
        }.split(DELIMITER)
    }
```

Reply:
오... `readln`, `ifBlank` 라는 함수가 있는지 몰랐네요
알려주셔서 감사합니다!